hosts.go

```diff
@@ -11,19 +11,19 @@ import (
 
 // Host host information
 type Host struct {
-	ID               string      `json:"id,omitempty"`
-	Name             string      `json:"name,omitempty"`
+	ID               string      `json:"id"`
+	Name             string      `json:"name"`
 	DisplayName      string      `json:"displayName,omitempty"`
 	CustomIdentifier string      `json:"customIdentifier,omitempty"`
-	Type             string      `json:"type,omitempty"`
-	Status           string      `json:"status,omitempty"`
-	Memo             string      `json:"memo,omitempty"`
-	Roles            Roles       `json:"roles,omitempty"`
+	Type             string      `json:"type"`
+	Status           string      `json:"status"`
+	Memo             string      `json:"memo"`
+	Roles            Roles       `json:"roles"`
 	RoleFullnames    []string    `json:"roleFullnames,omitempty"`
```

Comment:
(I think This `RoleFullnames` field in `Host` struct is never filled, [as documented in API docs](https://mackerel.io/api-docs/entry/hosts#get). `roleFullnames` only exists in request params of creating/updating host.)